Description: Do not test the version of glpk
 The test should not fail just because the version is different.
Author: Tobias Hansen <thansen@debian.org>
Forwarded: https://trac.sagemath.org/ticket/22443

--- a/sage/src/sage/numerical/mip.pyx
+++ b/sage/src/sage/numerical/mip.pyx
@@ -2519,7 +2519,7 @@
             sage: b.solver_parameter("simplex_or_intopt", "simplex_only")
             sage: b.solver_parameter("verbosity_simplex", "GLP_MSG_ALL")
             sage: p.solve()  # rel tol 1e-5
-            GLPK Simplex Optimizer, v4.60
+            GLPK Simplex Optimizer, v...
             2 rows, 2 columns, 4 non-zeros
             *     0: obj =   7.000000000e+00 inf =   0.000e+00 (2)
             *     2: obj =   9.400000000e+00 inf =   0.000e+00 (0)
